### ISSUE TITLE
Is there a library?

### ISSUE THREAD

**Issue by WarmSnowy** [google/fscrypt#265]
Hello, I want to develop a gui tools using fscrypt, is there a library I can use directly?

**Comment by ebiggers** [comment]
No, not currently.  It would be possible to use the `actions` Go package directly, although this isn't guaranteed to be stable from one version of `fscrypt` to the next.
